src/bmark/bmark_solvers.jl

```diff
@@ -37,20 +39,32 @@ if Pkg.installed("BenchmarkProfiles") != nothing
   * `stats`: a dict of statistics such as obtained from `bmark_solvers()`
 
   #### Keyword arguments
-  Any keyword argument accepted by `BenchmarkProfiles.performance_profile()`.
+  * `cost`: a function `cost(::ExecutionStats)` returning a positive cost value. Usually, time or number of function evaluations.
+  * Any keyword argument accepted by `BenchmarkProfiles.performance_profile()`.
 
   #### Return value
   A profile as returned by `performance_profile()`.
   """
-  function profile_solvers(stats :: Dict{Symbol, Array{Int,2}}; kwargs...)
+  function profile_solvers(stats :: Dict{Symbol, Array{ExecutionStats,1}};
+                           cost :: Function = stat->stat.elapsed_time,
+                           kwargs...)
     args = Dict(kwargs)
     if haskey(args, :title)
       args[:title] *= @sprintf(" (%d problems)", size(stats[first(keys(stats))], 1))
     end
-    performance_profile(hcat([sum(p, 2) for p in values(stats)]...),
-                        collect(String, [string(s) for s in keys(stats)]); args...)
+    solvers = collect(keys(stats))
+    np, ns = length(stats[solvers[1]]), length(solvers)
+    P = [stats[s][p].solved ? cost(stats[s][p]) : -1 for p = 1:np, s in solvers]
```

Comment:
No, because it's used on `performance_profile`. It would need some (apparently heavy) changes to make it work.